recipes/gsl-lite/config.yml

```diff
@@ -1,15 +1,19 @@
 versions:
+  "1.0.1":
+    folder: 1.x
+  "0.43.0":
+    folder: old
   "0.42.0":
-    folder: all
+    folder: old
   "0.41.0":
-    folder: all
+    folder: old
   "0.40.0":
-    folder: all
+    folder: old
   "0.39.0":
-    folder: all
+    folder: old
   "0.38.1":
-    folder: all
+    folder: old
   "0.38.0":
-    folder: all
+    folder: old
   "0.37.0":
-    folder: all
+    folder: old
```

Comment:
We could also remove all those versions (`0.42.0` to `0.37.0`) from here to stop publishing new revisions for them.